infra/Makefile.toml

```
@@ -5,18 +5,36 @@ extend = [
   { path = "scripts/ethapi.toml" },
   { path = "scripts/node.toml" },
   { path = "scripts/testnet.toml" },
-  { path = "scripts/testnode.toml" }
+  { path = "scripts/testnode.toml" },
+  { path = "scripts/subnet.toml" }
 ]
 
 [config]
 default_to_workspace = false
 
 [env]
-CHAIN_NAME = { value = "root", condition = { env_not_set = ["CHAIN_NAME"] }}
+# General network-specific parameters
+SUBNET_ID = { value = "root", condition = { env_not_set = ["SUBNET_ID"] }}
```

```suggestion
SUBNET_ID = { value = "/r0", condition = { env_not_set = ["SUBNET_ID"] }}
```

That's the default we use [here](https://github.com/consensus-shipyard/fendermint/blob/66ca42d08c1708c99ea4d0444d9d2ac24a0516d8/fendermint/app/config/default.toml#L97). My understanding is all subnet IDs start with `/r` and 0 means it's default, unspecified.

---

Correct. Good catch, Denis was allowing any chain name and completely missed this one. Thanks!